### DISCUSSION TITLE
Changing the color of the background for EPUB files?

### DISCUSSION THREAD

**Discussion by tor-soft** [edrlab/thorium-reader#2062] [Q&A]
Currently the only themes are Neutral, Sepia, and Night. Is there a way to change the background color to be a custom color (namely gray)? Im currently using it on Windows, installed using the setup executable.

**Comment by danielweck** [comment]
Hello, thank you for your feedback. I am afraid this feature is not available yet, but we have been discussing the technicalities of "custom colour themes". I think the topmost priority is to get "dark mode" implemented correctly, taking into account Windows high-contrast modes as well (accessibility). I believe Thorium will eventually offer theme presets in addition to the existing "sepia" look, as well as a user interface to allow users to create new arbitrary combination of foreground / background colours. That being said, we are not able to provide an estimated timeframe due to other development priorities at the moment.

**Reply by danielweck** [reply]
related issues https://github.com/edrlab/thorium-reader/issues?q=is%3Aissue+is%3Aopen+themes+
